### PR TITLE
Feature: Cat list

### PR DESCRIPTION
**Pull request by JanaS87**
each cat is listed as card displays, clearly legible, the names.
Styling is not final.

Help, tips or suggestions are always welcome!

### CODE REVIEW

File: components/CatList/CatList.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { cats } from "../../data/catdata";
+import CatCard from "../CatCard/CatCard.js";
+import { styled } from "styled-components";
+
+export default function CatList() {
+  return (
+    <>
+      <StyledHeading>Your Cats🐾</StyledHeading>
+      <StyledList>
+        {cats &&
```
Comment (jonas8900) [inline components/CatList/CatList.js:10]: could you explain why you use the && operator here? Maybe i understood the operator wrong

File: pages/index.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
+import CatList from "@/components/CatList/CatList";
+
 export default function HomePage() {
   return (
     <div>
```
Comment (jonas8900) [inline pages/index.js:5]: better use fragments here to not implement a div to the DOM tree

File: components/CatList/CatList.js (hunk 1 above)
Comment (JanaS87) [reply components/CatList/CatList.js:10]: first i have to check if the array is filled at all, if yes then i can map the list and display each cat as a card

File: pages/index.js (hunk 2 above)
Comment (JanaS87) [reply pages/index.js:5]: thanks for your suggestion, I will change this :)

File: components/CatCard/CatCard.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export default function CatCard({ cat }) {
+  return (
+    <>
```
Comment (jonas8900) [inline components/CatCard/CatCard.js:3]: you don't need the fragments for a single h3 but i think you'll fill the CatCard maybe in the next user story but I just wanted to say that you don't need these yet

File: components/CatList/CatList.js (hunk 1 above)
Comment (jonas8900) [reply components/CatList/CatList.js:10]: ahhhh thank you !

Comment (jonas8900) [reply components/CatList/CatList.js:10]: smart

File: pages/index.js (hunk 2 above)
Comment (JanaS87) [reply pages/index.js]: changes are committed

File: components/CatCard/CatCard.js (hunk 3 above)
Comment (JanaS87) [reply components/CatCard/CatCard.js]: changes are committed

**Pull request merged**